packages/js/src/preferences/preferences.ts

```diff
@@ -62,10 +64,4 @@ export class Preferences extends BaseModule {
       }
     });
   }
-
-  async update(args: UpdatePreferencesArgs): Result<Preference> {
-    return this.callWithSession(async () =>
-      updatePreference({ emitter: this._emitter, apiService: this._inboxService, args })
-    );
-  }
```

Comment:
This `update` method on the full `Preferences` resource was never used, so I'm deleting it.